spec/models/form_profile_spec.rb

```diff
@@ -1035,17 +1032,6 @@ def stub_methods_for_emis_data
         end
       end
 
-      context 'with emis and vet360 prefill for 0873' do
```

Comment:
Same as previous, since there is still a model for form_profile va_0873, this should probably be left in.